waterdata/tests/test_location.py

```diff
@@ -0,0 +1,240 @@
+"""
+Unit tests for waterdata.waterservices classes and functions.
+
+"""
+import datetime
+from unittest import TestCase, mock
+
+import requests as r
+
+from ..location import Parameter, MonitoringLocation
+
+
+class TestMonitoringLocation(TestCase):
+
+    def setUp(self):
+        # an actual test record may include many more columns from an RDB file record
+        self.test_record = {'site_no': '01630500',
+                            'agency_cd': 'USGS',
+                            'station_nm': 'POTOMAC RIVER NEAR WASH, DC LITTLE FALLS PUMP STA',
+                            'dec_lat_va': '38.94977778',
+                            'dec_long_va': '-77.12763889'
+                            }
+        self.test_rdb_text = ('#\nagency_cd\tsite_no\tstation_nm\tsite_tp_cd\tdec_lat_va\tdec_long_va\tcoord_acy_cd\t'
```

Comment:
Personal preference - but for readability I'd prefer a multi-line """xxx""" style string.